### PR TITLE
Adding fs tc 83573836

### PR DESCRIPTION
**Pull request by julpark-rh**
CEPH-83573837 - Verify the performance stats of the filesyetem

https://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-Q7RUHW

# Description

Please include Automation development guidelines. Source of Test case - New Feature/Regression Test/Close loop of customer BZs
<details>

<summary>click to expand checklist</summary>

- [ ] Create a test case in Polarion reviewed and approved.
- [ ] Create a design/automation approach doc. Optional for tests with similar tests already automated.
- [ ] Review the automation design
- [ ] Implement the test script and perform test runs
- [ ] Submit PR for code review and approve
- [ ] Update Polarion Test with Automation script details and update automation fields
- [ ] If automation is part of Close loop, update BZ flag qe-test_coverage “+” and link Polarion test
</details>

### CODE REVIEW

**Review by hkadam134 — APPROVED** [review]
Looks Good

File: tests/cephfs/ceph_83573836.py
Comment (hkadam134) [inline tests/cephfs/ceph_83573836.py:1]: Change the file name to match the TC summary.

**Comment by hkadam134** [comment]
Can you create a suite file for cephfs-top ? -  "tier-3_cephfs_test-cephfs-top.yaml"

**Review by AmarnatReddy — APPROVED** [review]
LGTM

**Pull request merged**